core/domain/html_validation_service.py

```diff
@@ -957,24 +957,39 @@ def get_filename_with_dimensions(exploration_fs, old_filename):
     Args:
         exploration_fs: GcsFileSystem. File system of the Exploration entity.
         old_filename: str. Name of the file whose dimensions need to be
-            calculated.
+            calculated. The filename is not expected to contain the pattern
```

Comment:
is not expected to --> should not

contain the pattern --> end with the pattern